tests/jballerina-unit-test/src/test/resources/test-src/object/object_constructor.bal

```diff
@@ -12,3 +12,24 @@ public function testObjectInit() returns int {
     p = new;
     return a;
 }
+
+function testLocalVariablesAssignmentToFields() {
+    int a = 10;
+
+    var obj = object {
```

Comment:
Let's add tests for module-level object constructors also.

Reply:
Added